compiler/damlc/BUILD.bazel

```diff
@@ -80,6 +82,8 @@ package_app(
     resources = [
         ":ghc-pkg-dist",
         "//compiler/damlc/daml-ide-core:dlint.yaml",
+        "//compiler/damlc/daml-visual:d3.js",
```

Comment:
Have you checked that it actually works in `daml-sdk-head`? iirc these files will end up in `resources/d3.js` so your `readFile` call is definitely not going to work. `locateRunfiles` should do the right thing™ if you call it on the directory and then read `dir </> filename` where `dir` is the result from `locateRunfiles`.

Reply:
I did not check with `daml-head` but in any case, I am going to make changes to make use of `locateRunfiles`

Reply:
It would be good to have an integration test in `//daml-assistant/integration-tests` that verifies that this works if you run it from an SDK installation as well.

Reply:
about the resources, damlc `package_app` is not packaging up `"@static_asset_d3plus//:d3-assets",` Any ides what that could be happening ?
```
➜  quickstart ls -l ~/.daml/sdk/0.0.0/damlc/resources/
total 67536
-r--r--r--   1 anupkalburgi  staff     48519 Sep 25 15:02 dlint.yaml
-r-xr-xr-x   1 anupkalburgi  staff    534668 Aug 19 09:25 ghc-pkg
dr-xr-xr-x  30 anupkalburgi  staff       960 Sep 25 15:52 lib
dr-xr-xr-x   4 anupkalburgi  staff       128 Sep 25 15:52 pkg-db_dir
-r-xr-xr-x   1 anupkalburgi  staff  33992088 Sep 25 15:11 scenario-service.jar
➜  quickstart
```

Reply:
I’m not sure filegroups work properly, maybe try changing it to use `exports_files` and then reference both files explicitly.

Reply:
Export files it is !